src/main/java/edu/greenblitz/robotName/subsystems/swerve/modules/mk4iSwerveModule/MK4iSwerveConstants.java

```diff
@@ -21,13 +22,13 @@ public class MK4iSwerveConstants {
 	
 	public static final double ANGULAR_GEAR_RATIO = (150.0 / 7);
 	
-	public static final double LINEAR_GEAR_RATIO = 8.14;
+	public static final double LINEAR_GEAR_RATIO = 6.12;
 
 	public static final double ks = 0.32;
 
-	public static final double kv = 2.5289;
+	public static final double kv = 0;
```

Comment:
What

Reply:
we are doing position control on the angular swerve motor so we dont need kv